docs/2.5. APIs - Query Parameters.md

```diff
@@ -20,7 +20,7 @@ The following implementation notes should be observed:
 
 * The choice to page based on 'creation' or 'update' timestamp depends on the client's intended use of the data. Paging based on 'creation' time from the start of a collection provides the best mechanism to identify all resources held in a registry. Paging based on 'update' time from the end of a collection provides the best mechanism to watch for changing resources in the case that Websocket subscriptions are not available.
 
-* Paging through a collection for a second time, using the same cursors as previously should never result in new data appearing in the response payloads. However, note that as the registry is dynamic and resources may be updated or deleted a given cursor may return fewer results than it did previously.
+* Paging through a collection for a second time, using the same cursors as previously should never result in new data appearing in the response payloads when using 'creation' ordering. However, note that as the registry is dynamic and resources may be updated or deleted a given cursor may return fewer results than it did previously. When using 'update' ordering and a query filter, new data may appear when cursors are used for a second time.
```

Comment:
Does the (contrived?) example of sending a request anchored to a future time like `paging.until=2345678901:0` need calling out as possibly returning new resources, or does "using the same cursors as previously" make clear its the returned `X-Paging-Since/Until` values that are being referred to?